src/server/game/Guilds/GuildMgr.h

```diff
@@ -47,6 +47,9 @@ class TC_GAME_API GuildMgr
     Guild* GetGuildByName(std::string_view guildName) const;
     std::string GetGuildNameById(ObjectGuid::LowType guildId) const;
 
+    // --- new read-only accessor ---
```

Comment:
This comment is written as if it was meant for the person reviewing the PR, remove it